spring-cloud-gcp-samples/spring-cloud-gcp-config-sample/pom.xml

```diff
@@ -1,65 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
-
-    <parent>
-        <!-- Your own application should inherit from spring-boot-starter-parent -->
-        <artifactId>spring-cloud-gcp-samples</artifactId>
-        <groupId>com.google.cloud</groupId>
-				<version>5.9.1-SNAPSHOT</version><!-- {x-version-update:spring-cloud-gcp:current} -->
-    </parent>
-
-    <modelVersion>4.0.0</modelVersion>
-    <artifactId>spring-cloud-gcp-config-sample</artifactId>
```

Comment:
You mention in the PR description:
> spring-cloud-gcp-config-sample (breaking change): removed autoconfiguration and sample code. This is to adjust to what's stated in [the docs](https://googlecloudplatform.github.io/spring-cloud-gcp/5.9.0/reference/html/index.html#cloud-runtime-configuration-api).

1. If we've chosen to completely remove the starter and sample, we should also remove the documentation on it. You'll find it @ https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/main/docs/src/main/asciidoc/config.adoc

2. Have we decided to completely remove the starter and sample? Could you create a tab in your migration doc that discusses why we've made this choice?

Reply:
Added a discussion tab